deploy/infraagent-configmap.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
 data:
   infraagent.yaml: |
     interfaceType: tap
+    insecure: true
```

Comment:
Are we defaulting to insecure mode? I don't think that's acceptable from SDLe perspective.

Reply:
As mentioned above comment this is temporary config. (it was described in the [commit message](https://github.com/ipdk-io/k8s-infra-offload/pull/37/commits/164ad0f003e5a759a48694f734df52d1e8356c24) as well )

Reply:
ok, I missed the commit message. As long as we don't make a release with this default setting, we should be okay.